GVRf/Framework/framework/src/main/jni/objects/bounding_volume.cpp

```diff
@@ -182,6 +182,9 @@ void BoundingVolume::transform(const BoundingVolume &in_volume,
             max_corner_.z += a;
         }
     }
+    // expand with the new corners
+    expand(min_corner_);
```

Comment:
I see. I assumed the bounding radius was not used.

In any case you could add the following lines in place of expand(), will eliminate the need for the 6 if conditionals in the expand method

```
    center_ = (min_corner_ + max_corner_) * 0.5f;
    if (min_corner_ == max_corner_)
    	radius_ = 0;
    else
    	radius_ = glm::length(max_corner_ - min_corner_) * 0.5f;
```